modules/windows-node-services.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/understanding-windows-containers.adoc
+
+[id="windows-node-services_{context}"]
+= Windows node services
+
+The following Windows-specific services are installed on each Windows node:
+
+[cols="1,2",options="header"]
+|===
+
+|Service
+|Description
+
+|kubelet
+|Registers the Windows node and manages its status.
+
+|Container Network Interface (CNI) plugins
+|Exposes link:https://kubernetes.cn/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#networking[networking] for Windows nodes.
```

Comment:
I am having trouble connecting to the site. Not sure if this is a local issue.

Reply:
I think the TLD is wrong, works for with `io`: https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#networking

Reply:
Thanks; I'll update that!